wagtail/admin/rich_text/converters/html_to_contentstate.py

```diff
@@ -12,7 +12,11 @@
 KEEP_WHITESPACE = 1
 FORCE_WHITESPACE = 2
 
-WHITESPACE_RE = re.compile(r'\s+')
+# match more than one consecutive normal spaces, new-lines, tabs and form-feeds
+WHITESPACE_RE = re.compile(r'[ \t\n\f]{2,}')
+
+# match more than one consecutive zero-width spaces
+ZERO_WIDTH_SPACE_RE = re.compile(r'[\u200b]{2,}')
```

Comment:
What's the reason for collapsing zero-width spaces? Is this something that happens in real-world usage (for example, when pasting content from a Word document)?

I can't see any good reason for a document to have multiple zero-width spaces, but I can't see why they would cause harm either - and in this case I'd prefer not to make non-essential changes.